app/scripts/controllers/client/ViewClientController.js

```diff
@@ -846,31 +846,85 @@
                 $scope.cancel = function () {
                     $uibModalInstance.dismiss('cancel');
                 };
+                $scope.uploadSig = function (file) {
```

Comment:
Make the same changes as suggested in the deleteSig, just make this function open up the uploading modal where the user can select the file to upload and confirm

Reply:
Remove the `file` argument it is not needed